src/site/tools/pub/cmd/pub-downgrade.markdown

```diff
@@ -56,7 +56,7 @@ Resolving dependencies...
   source_span 1.0.2
 < stack_trace 0.9.2 (was 1.1.1)
   stagexl 0.10.2
-< unittest 0.10.0 (was 0.11.4)
+< test 0.10.0 (was 0.11.4)
```

Comment:
It'd be nice to use real, more up-to-date version #s. Perhaps run pub upgrade on one of our samples that uses test, and then pub downgrade.